UPGRADE-4.2.md

```diff
@@ -285,6 +285,7 @@ Messenger
    ```
  * The `EncoderInterface` and `DecoderInterface` interfaces have been replaced by a unified `Symfony\Component\Messenger\Transport\Serialization\SerializerInterface`.
    Each interface method have been merged untouched into the `Serializer` interface, so you can simply merge your two implementations together and implement the new interface.
+ * The `HandlerLocator` class was replaced with `Symfony\Component\Messenger\Handler\HandlersLocator` where also the constructor changes instead of a one to one relation between message and handler its now needed to define it as one to many relation between message to handlers.
```

Comment:
i suggest the a before/after example then, as this sentence is really hard to understand :}

Reply:
@ro0NL updated is it ok so?